daml-lf/archive/src/main/protobuf/com/daml/daml_lf_dev/daml_lf_1.proto

```diff
@@ -568,7 +568,9 @@ enum BuiltinFunction {
   NUMERIC_TO_BIGNUMERIC = 145; // *Available in versions >= 1.13*
   BIGNUMERIC_TO_TEXT = 146; // *Available in versions >= 1.13*
 
-  // Next id is 148. 147 is ANY_EXCEPTION_MESSAGE.
+  TYPEREP_TYCON_NAME = 148; // *Available in versions >= 1.dev*
+
+  // Next id is 149.
```

Comment:
```suggestion
  // Next id is 149. 148 is TYPEREP_TYCON_NAME.
```

Reply:
Added it; I'd left it out since 147 was in a different part of the file